packages/dx-react-chart-bootstrap4/src/templates/series/point.jsx

```diff
@@ -25,9 +19,11 @@ Point.propTypes = {
   x: PropTypes.number.isRequired,
   y: PropTypes.number.isRequired,
   d: PropTypes.string.isRequired,
+  pathComponent: PropTypes.any,
```

Comment:
Is it really `PropTypes.any`? Maybe:

```js
pathComponent: PropTypes.func,
```

Reply:
Yes, it's the workaround for react warnings. We will fix it in the feature.